chapters/clears.txt

```diff
@@ -154,6 +154,8 @@ endif::VK_VERSION_1_1,VK_KHR_maintenance1[]
   * [[VUID-vkCmdClearDepthStencilImage-image-00010]]
     If pname:image is non-sparse then it must: be bound completely and
     contiguously to a single sname:VkDeviceMemory object
+  * [[VUID-vkCmdClearDepthStencilImage-image-00014]]
+    pname:image must: have a depth/stencil format
```

Comment:
I'm not clear why you moved this VU? Just seems like churn to me?

Reply:
Probably esthetics. Seems better with the other `image` VUs, and before all the longwinded VUs...

Reply:
In general I'd rather not move individual VUs around outside of a general "tidy up" pass, because it just means more chance for conflicts with other in-flight proposals.

Reply:
As you wish. Though having barriers against tidying usually means it is virtually never done...